File: src/main/java/com/aquasecurity/plugins/trivy/model/IacMetadata.java
package com.aquasecurity.plugins.trivy.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class IacMetadata {
    @JsonProperty("Resource")
    public String resource;
    @JsonProperty("Provider")
    public String provider;
    @JsonProperty("Service")
    public String service;
    @JsonProperty("StartLine")
    public int startLine;
    @JsonProperty("EndLine")
    public int endLine;
}

File: src/main/kotlin/com/aquasecurity/plugins/trivy/model/IacMetadata.kt
package com.aquasecurity.plugins.trivy.model

import com.fasterxml.jackson.annotation.JsonProperty

class IacMetadata {
    @JsonProperty("Resource")
    var resource: String? = null

    @JsonProperty("Provider")
    var provider: String? = null

    @JsonProperty("Service")
    var service: String? = null

    @JsonProperty("StartLine")
    var startLine: Int = 0

    @JsonProperty("EndLine")
    var endLine: Int = 0
}